service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -328,15 +369,11 @@ public void channelInactive(ChannelHandlerContext ctx) {
                 isManualDisconnect = false;
             } else {
                 super.channelInactive(ctx);
-                LOG.info("Reopening websocket because it was closed by the host");
-                final Completable c = connect()
-                        .doOnError(t -> LOG.warn("Problem with reconnect", t))
-                        .retryWhen(new RetryWithDelay(retryDuration.toMillis()))
-                        .doOnComplete(() -> {
-                            LOG.info("Resubscribing channels");
-                            resubscribeChannels();
-                        });
-                c.subscribe();
+                if (conenctedSuccessfully) {
```

Comment:
This conflicts with #191.  It looks like #191 attempts to fix the reconnect but you've removed it entirely.

However, I'm pretty new to this project and still waiting for confirmation

Reply:
I moved it to  'connect' method